src/Resource/Payment.php

```diff
@@ -593,36 +604,52 @@ public function setEscrow($description)
      */
     public function capture()
     {
-        if (is_null($this->order)) {
-            throw new \Exception('Sorry, multipayment capture is not available on this version');
+        if (!$this->isMultipayment($this->getId())) {
```

Comment:
Não precisa do else ai faz assim: 
`
$path = sprintf('/%s/%s/%s/%s', MoipResource::VERSION, self::MULTI_PAYMENTS_PATH, $this->getId(), 'capture');
 if (!$this->isMultipayment($this->getId())) {
$path = sprintf('/%s/%s/%s/%s', MoipResource::VERSION, self::PATH, $this->getId(), 'capture');
}
`